packages/slice-machine/components/ScreenshotPreviewModal/index.tsx

```diff
@@ -0,0 +1,115 @@
+import SliceMachineModal from "@components/SliceMachineModal";
+import { useSelector } from "react-redux";
+import { SliceMachineStoreType } from "@src/redux/type";
+import { isModalOpen } from "@src/modules/modal";
+import { ModalKeysEnum } from "@src/modules/modal/types";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import { Button, Close, Container, Flex, Heading, Image } from "theme-ui";
+import Card from "@components/Card";
+
+type ScreenshotModalProps = {
+  sliceName: string;
+  screenshotUrl: string;
+};
+
+const ScreenshotPreviewModal: React.FunctionComponent<ScreenshotModalProps> = ({
```

Comment:
OPT: put a small comment to describe where the component is used.
It's name being similar to the "ScreenshotChangesModal", this could help someone that doesn't know the code to get more context.